packages/core/src/browser/shell/tab-bars.ts

```diff
@@ -131,8 +132,9 @@ export class TabBarRenderer extends TabBar.Renderer {
     /**
      * If size information is available for the label, set it as inline style. Tab padding
      * and icon size are also considered in the `top` position.
+     * @param isInSidePanel {boolean} an optional check which determines if the tab is in the side-panel.
```

Comment:
I think we can skip on the typing/brackets here, since it is already declared in the implementation. Documentation is good, just trying to avoid being redundant :)